### ISSUE TITLE
Big Bird: Transformers for Longer Sequences 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#4]
# TL;DR
A study that made Self-Attention more efficient. Combining three type attention: random, peripheral only, and full (with only some tokens). They showed that in many NLP tasks it is SOTA and theoretically an approximation of s2s and Turing completeness.
![image](https://user-images.githubusercontent.com/24524018/89171838-a8284200-d5bc-11ea-9bde-5e39129a25c8.png)

# Paper URL
https://arxiv.org/abs/2007.14062
# Submission Dates(yyyy/mm/dd)
2020/07/20

# Authors and institutions
Manzil Zaheer, Guru Guruganesh, Avinava Dubey, Joshua Ainslie, Chris Alberti, Santiago Ontanon, Philip Pham, Anirudh Ravula, Qifan Wang, Li Yang, Amr Ahmed
- Google Research
